new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWITValidation.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes;
+
+import oracle.weblogic.kubernetes.actions.TestActions;
+import oracle.weblogic.kubernetes.actions.impl.primitive.WITParams;
+import oracle.weblogic.kubernetes.actions.impl.primitive.WebLogicImageTool;
+import oracle.weblogic.kubernetes.extensions.LoggedTest;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@DisplayName("Simple validation of basic WIT functions")
+class ItWITValidation implements LoggedTest {
+  @Test
+  @DisplayName("Create a MII image")
+  public void testCreatingMIIImage() {
+
+    // install WIT using the default version and location
+    boolean downloadWIT = TestActions.installWIT(
+        null /* version */, 
+        null /* location */,
+        false /* do not redirect output */);
+   
+    assertEquals(true, downloadWIT);
+    
+    // install WDT using the default version and location
+    boolean downloadWDT = TestActions.installWDT(
+        null /* version */, 
+        null /* location */,
+        false /* do not redirect output */);
+
+    assertEquals(true, downloadWDT);
+
+    // create the MII image
+    // TODO add model files and other contents to the image once we have those resources
+    WITParams params = new WITParams()
+        .baseImageName(WebLogicImageTool.WLS_BASE_IMAGE_NAME)
```

Comment:
maybe these should just be defaults?
we could have a `defaults()` method on `WITParams` to set these? @rjeberhard what do you think?

Reply:
Makes sense to me, and would make these a little less verbose and easier to read.

Reply:
I have made the changes locally, and the code looks much better. Thanks.

Reply:
I also moved the constants in WebLogicImageTool into WITParams. I like it. 
The only thing that I don't like is that the test code still refers to the impl class WITParams. We'll have to introduce an interface to avoid that. Shallow do that? or any other suggestions?

Reply:
@ddsharpe suggested a withWITParams() or something like that, which can be static imported, and returns an instance, so you don't need to write/init the class name....

Reply:
Let me explore more. thanks.

Reply:
I got something working:). Will push soon.

Reply:
done

Reply:
made one more change for this. I am happy now:).